Source/UserManagement/UserManagement.code-workspace

```diff
@@ -0,0 +1,10 @@
+{
+	"folders": [
+		{
+			"path": "/Users/einari/Projects/RedCross/cbs/Source/Infrastructure"
```

Comment:
Not sure what this does, but the path is specific to your machine, can this be made generic?

Reply:
Weird that VSCode puts in absolute paths - I've fixed these in an upcoming PR to be relative.